mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -78,17 +78,17 @@ trait Smt {
   val prof: Boolean
 }
 
-// TODO: Move into "IO"
-trait Parser {
+trait Input {
   val source: SourceOption
 }
 
 /** Options used to configure program input and output */
-trait IO {
+trait Output {
   val output: Option[String] = None
-  val outdir: Path
```

Comment:
This is not a pass option, as it is only used during program initialization by the output manager, thus the removal here.